src/ui/public/vislib/__tests__/lib/layout/layout.js

```diff
@@ -12,6 +12,8 @@ import $ from 'jquery';
 import VislibLibLayoutLayoutProvider from 'ui/vislib/lib/layout/layout';
 import FixturesVislibVisFixtureProvider from 'fixtures/vislib/_vis_fixture';
 import PersistedStatePersistedStateProvider from 'ui/persisted_state/persisted_state';
+import VislibVisConfig from 'ui/vislib/lib/vis_config';
```

Comment:
in the tests i kept the paths absolute to prevent imports like '../lib/vis_config' ... what do you think ?